brooklyn-tosca-transformer/src/test/java/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaPlanToSpecTransformerTest.java

```diff
@@ -59,7 +65,7 @@ public void testSimpleHostedTopologyParser() {
     }
 
     // FIXME: Test fails when asserting the size of the tomcat server's config map.
-    @Test(enabled = false)
```

Comment:
Sorry. But, this test is running in my machine. What error did you find? I will try to solve it.